ksql-engine/src/main/java/io/confluent/ksql/util/KafkaTopicClientImpl.java

```diff
@@ -64,24 +66,31 @@ public KafkaTopicClientImpl(final AdminClient adminClient) {
   public void createTopic(
       final String topic,
       final int numPartitions,
-      final short replicatonFactor
+      final short replicatonFactor,
+      boolean isCompacted
```

Comment:
We should use the TopicCleanupPolicy here instead of isCompacted. Also, wouldn't it be better to use compact+delete for topics backing windowed aggregations?

Both of these changes can be made in a later PR targeting master